app.config.yaml

```diff
@@ -32,7 +32,7 @@ application:
             web: 'no'
             runtime: nodejs:22
             limits:
-              memorySize: 128
+              memorySize: 256
```

Comment:
default mem is 256 so no need to set it explicitly
but I think we can leave it here together with a note that customers should test their own scenario and increase if needed

Reply:
ok makes sense